### DISCUSSION TITLE
Working great on OrangePi Zero2

### DISCUSSION THREAD

**Discussion by prosper00** [atauenis/webone#53] [General]
This is an excellent tool for vintage computers! 

I successfully installed the .deb on Bullseye aarm64 on my OrangePi Zero2, it's working great. 

On my old RaspberryPi 1B, running DietPi (basically Bullseye 32-bit armhf), it just segfaults. Could just be an environment issue, I don't plan to debug (the pi1 is just so painfully SLOW to work with).

**Comment by chaoszone** [comment]
Are you running it headless? I am having difficulties letting my bash script which executes the "dotnet webone.dll" start automatically on boot.
